docs/wiki/boards/MULTIFLITEPICO.md

```diff
@@ -17,7 +17,7 @@ The features of the FC are detailed below:
 - MPU6050
 - Brushed motor support for 4 motors (all populated with fets)
 - M5 and M6 broken out (used when converting into a HEX)
-- 8A N Channel fets (10A <5secs)
+- 8A N Channel fets (10A \<5secs)
```

Comment:
Is `\` a escape character here to use `<` ?

Reply:
yes, brackets need to be escaped https://docusaurus.io/docs/migration/v3#bad-usage-of--1